client/src/pages/CalendarHomePage/CalendarHomePage.js

```diff
@@ -47,35 +74,48 @@ class CalendarHomePage extends Component {
       String(selectedDate.month()) + String(selectedDate.year());
 
     return (
-      <div className="calendar">
```

Comment:
Use https://github.com/codefordenver/Comrad/pull/479/files?w=1 to see the changes: this whole section was wrapped in a `<Card>` so indentation changed throughout the section.